downstream/modules/platform/proc-aap-controller-restore.adoc

```diff
@@ -0,0 +1,37 @@
+[id="aap-controller-restore"]
+
+= Recovering the {ControllerNameStart} deployment
+
+[role=_abstract]
+Use this procedure to restore a previous controller deployment from an AutomationControllerBackup. The deployment name you provide will be the name of the new AutomationController custom resource that will be created.
+
+[NOTE]
+====
+The name specified for the new AutomationController custom resource must not match an existing deployment or the recovery process will fail.
```

Comment:
I notice we have this noted twice in this section... But I'm not wholly against it. This is an important point I want to make sure customers see.